### PR TITLE
feat: add default tokens to swap

### PR DESCRIPTION
**Pull request by vvava**
## Description

We want to add default tokens to the Swap page

## Changes

Set the AVAX and USDC tokens as default transaction pairs.

## Testing

Go to the swap page -> you should see the AVAX in the `from` and the USDC in the `to` field -> swap should work as before

## Screenshots:

https://github.com/user-attachments/assets/0cc4998b-7627-4284-ad56-e807c69cd9ba

## Checklist for the author

Tick each of them when done or if not applicable.

- [ ] I've covered new/modified business logic with Jest test cases.
- [x] I've tested the changes myself before sending it to code review and QA.

### CODE REVIEW

File: src/pages/Swap/Swap.tsx [hunk 1]
```diff
@@ -104,6 +105,30 @@ export function Swap() {
     destAmount,
   } = useSwapStateFunctions();
 
+  useEffect(() => {
+    if (!selectedFromToken && !selectedToToken) {
```
Comment (meeh0w) [inline src/pages/Swap/Swap.tsx:109]: This check alone does not quite work, tokens are not restored when I close and re-open extension.

### Prod build:

https://github.com/user-attachments/assets/11e866de-ffec-499f-8af0-28aa140a065c


### This branch:

https://github.com/user-attachments/assets/b4f6887e-925c-4b59-87f2-b14a2eb94d14

**Review by meeh0w — CHANGES_REQUESTED** [review]

**Review by meeh0w — CHANGES_REQUESTED** [review]
There is still something wrong with remembering the values. It sometimes works and sometimes doesn't. There's probably a race condition somewhere.

https://github.com/user-attachments/assets/24f14d30-a466-4cc4-9480-850f3a19c74b

**Comment by vvava** [comment]
> There is still something wrong with remembering the values. It sometimes works and sometimes doesn't. There's probably a race condition somewhere.
> 
>  Screen.Recording.2024-10-31.at.11.53.19.mov

Yeah, something is wrong there, I'm fixing it

**Review by meeh0w — APPROVED** [review]
Looks like it's working now 👏

**Review by bferenc — APPROVED** [review]

**Pull request merged**